dev/breeze/src/airflow_breeze/utils.py

```diff
@@ -0,0 +1,41 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import hashlib
+import shlex
+import subprocess
+from typing import List
+
+
+def run_command(cmd: List[str], *, check: bool = True, **kwargs):
+    print(f"$ {' '.join(shlex.quote(c) for c in cmd)}")
+    try:
+        return subprocess.run(cmd, check=check, **kwargs)
+    except subprocess.CalledProcessError as ex:
+        print("========================= OUTPUT start ============================")
+        print(ex.stderr)
+        print(ex.stdout)
+        print("========================= OUTPUT end ============================")
+        raise
+
+
+def generate_md5(filename, file_size: int = 65536):
```

Comment:
I think we do not need to limit the size. We run those md5 sum on  files that will never grow too large but they might get bigger than 64K in extreme cases (Dockerfiles)

Reply:
I have set the default one to 64K, but we could pass the file_size in any case to override the 64K. Do you have any suggestion for me to make this better?

Reply:
Ah... OK. My bad. I have not seen the iteration below. It's ok as it is now!

Reply:
I thought it's only 64K that are used, but now I noticed you iterated whole file with 64K chunks, which is a very good idea.